examples/get-started/contour-map.py

```diff
@@ -0,0 +1,130 @@
+"""
+2. Create a contour map
+=======================
+
+This tutorial page covers the basics of creating a figure of Earth relief,
+using a remote dataset hosted by GMT, using the method
+:meth:`pygmt.datasets.load_earth_relief`. It will use
+:meth:`pygmt.Figure.grdimage`, :meth:`pygmt.Figure.grdcontour`,
+:meth:`pygmt.Figure.colorbar`, and :meth:`pygmt.Figure.coast` methods for
+plotting.
+"""
+
+# sphinx_gallery_thumbnail_number = 4
+import pygmt
+
+###############################################################################
+# Loading the Earth relief dataset
+# --------------------------------
+#
+# The first step is to import :meth:`pygmt.datasets.load_earth_relief`.
+# The ``resolution`` parameter sets the resolution of the remote grid file,
+# which will affect the resolution of the plot made later in the tutorial.
+# The ``registration`` parameter determines the grid registration.
+#
+# This grid region covers the islands of Guam and Rota in the western Pacific
+# Ocean.
+
+grid = pygmt.datasets.load_earth_relief(
+    resolution="30s", region=[144.5, 145.5, 13, 14.5], registration="gridline"
+)
```

Comment:
<img width="915" alt="image" src="https://user-images.githubusercontent.com/3974108/195765876-22bdc613-7473-4af1-bac6-da6f0c62e527.png">

It seems this tile is not cached in the CI. Need to cache it in the CI or use another dataset that has been cached.